### ISSUE TITLE
Updating to 3.x

### ISSUE THREAD

**Issue by SimonThordal** [Graphify/graphify#31]
I can see that there hasn't been so much activity lately, but are there any plans of updating graphify to Neo4j 3.x?

**Comment by cbernardone** [comment]
Thanks for asking this. Were you even able to get it working with the older version of Neo4j? The earliest I can download seems to be 2.3.11 and it's giving me trouble working with graphify.

**Comment by SimonThordal** [comment]
Nope, didn't try it on older versions. The latest compiled binary seems to be for 2.2.1 though

**Comment by cbernardone** [comment]
Okay, thanks. A shame it's not been updated looks like a nice package.
